tornettools/simulate.py

```diff
@@ -32,6 +32,9 @@ def run(args):
 
     if comproc != None:
         logging.info(f"Done simulating; shadow returned code '{comproc.returncode}'")
+        return comproc.returncode
+
+    return 1
```

Comment:
> Not clear how this would happen. Log a warning or error if it does?

It looks like tornettool's `__run_shadow()` sometimes returns `None` and logs a warning to indicate an error rather than using an exception, so here it means there was an error before starting shadow.

> Also a bit of a stylistic nit, but IMO it's clearer to check for error and exit abnormally, keeping the "error paths" more clearly conditional. i.e.

Agreed, I changed this to check the error condition first.